src/code/z_play_hireso.c

```diff
@@ -1,27 +1,1360 @@
+        BOMBERS_NOTEBOOK_ENTRY_END,
+    },
+    {
+        /* Shiro */
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 1, BOMBERS_NOTEBOOK_EVENT_RECEIVED_STONE_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 2, BOMBERS_NOTEBOOK_EVENT_RECEIVED_STONE_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 3, BOMBERS_NOTEBOOK_EVENT_RECEIVED_STONE_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY_END,
+    },
+    {
+        /* Guru-Guru */
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 1, BOMBERS_NOTEBOOK_EVENT_RECEIVED_BREMEN_MASK,
+                               CLOCK_TIME(18, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 2, BOMBERS_NOTEBOOK_EVENT_RECEIVED_BREMEN_MASK,
+                               CLOCK_TIME(18, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY_END,
+    },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Draw.s")
+s16 sBombersNotebookDayRectRectLeft[] = { 120, 120, 270, 420 };
+TexturePtr sBombersNotebookDayTextures[] = {
+    gBombersNotebookDay1stTex,
+    gBombersNotebookDay2ndTex,
+    gBombersNotebookDayFinalTex,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016F4EC.s")
+#define DEFINE_EVENT(_enum, icon, _colorFlag, _description, _completedMessage, _completedFlag) icon,
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Update.s")
+s32 sBombersNotebookEventIcons[] = {
+#include "tables/bombers_notebook/event_table.h"
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Init.s")
+#undef DEFINE_EVENT
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Destroy.s")
+s32 sBombersNotebookEventIconWidths[] = { 16, 24, 32 };
+s32 sBombersNotebookEventIconHeights[] = { 16, 28, 28 };
+
+#define DEFINE_EVENT(_enum, _icon, colorFlag, _description, _completedMessage, _completedFlag) colorFlag,
+
+u16 sBombersNotebookEventColorWeekEventFlags[] = {
+#include "tables/bombers_notebook/event_table.h"
+};
+
+#undef DEFINE_EVENT
+
+void BombersNotebook_TextureRectangle(Gfx** gfxP, s32 rxl, s32 ryl, s32 rxh, s32 ryh, s32 tile, s32 s, s32 t, s32 dsdx,
+                                      s32 dtdy) {
+    s32 xl = rxl - (D_801FBBD0 * 4);
+    s32 yl = ryl - (D_801FBBD2 * 4);
+    s32 xh = rxh - (D_801FBBD0 * 4);
+    s32 yh = ryh - (D_801FBBD2 * 4);
+    Gfx* gfx = *gfxP;
+
+    gSPScisTextureRectangle(gfx++, xl, yl, xh, yh, tile, s, t, dsdx, dtdy);
+
+    *gfxP = gfx;
+}
+
+s16 sBombersNotebookHeaderColors[][3] = {
+    /* Column 1 */ { 116, 134, 146 },
+    /* Column 2 */ { 158, 156, 131 },
+    /* Column 3 */ { 174, 141, 151 },
```

Comment:
```suggestion
    { 116, 134, 146 }, // Column 1
    { 158, 156, 131 }, // Column 2
    { 174, 141, 151 }, // Column 3
```
Any reason for having them on the left? (Same with `sBombersNotebookColumnColors`)